File: AV.Common/Entities/Product.cs
using AV.Contracts.Enums;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;


namespace AV.Common.Entities
{
    public class Product
    {
        [Key]
        public int ProductId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public ProductType Type { get; set; }
        public ProductCategory Category { get; set; }
        public decimal? Price { get; set; }
        public string ServiceType { get; set; }
        public string SampleUrl { get; set; }
        public IEnumerable<ProductFeature> Features { get; set; }
    }
}

File: GoSmartValue.Web/ClientApp/src/app/shared/models/product.ts
import { IProduct} from "./interfaces/product";

export class Product implements IProduct{
  "productId" = 0;
  "name"= "";
  "description"= "";
  "type"= 0;
  "category"= 1;
  "price"= 0;
  "serviceType"= "";
  "sampleUrl"= "";
  "features": [
    {
      "id": 0;
      "pRoductId": 0;
      "title": "";
      "description": "";
      "position": 0
    }
  ]
}
